sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -238,6 +239,9 @@ func (m *marshaller) makePayload(edge *graph.MultiEdge) *pb.FunctionSpec {
 	case graph.GBK:
 		return &pb.FunctionSpec{Urn: urnGBK}
 
+	case graph.External:
+		return &pb.FunctionSpec{Urn: urnExternal}
```

Comment:
Should be: return &pb.FunctionSpec{Urn: edge.Payload.URN, Payload: edge.Payload.Data}

Reply:
Done.